server/driver/wivrn_pico_face_tracker.h

```diff
@@ -0,0 +1,55 @@
+/*
+ * WiVRn VR streaming
+ * Copyright (C) 2024  Guillaume Meunier <guillaume.meunier@centraliens.net>
+ * Copyright (C) 2024  Patrick Nicolas <patricknicolas@laposte.net>
+ * Copyright (C) 2024  galister <galister@librevr.org>
+ * Copyright (C) 2025  Sapphire <imsapphire0@gmail.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "wivrn_fb_face2_tracker.h"
+#include "wivrn_packets.h"
+#include "xr/pico_eye_types.h"
+#include "xrt/xrt_defines.h"
+#include "xrt/xrt_device.h"
+
+#include "history.h"
+
+#include <array>
+#include <cmath>
+#include <openxr/openxr.h>
```

Comment:
Not all includes seem to be needed in the header file.

Only keep what is directly used in this file, move to the cpp what is only used in the implementation.